EDITORS_GUIDE.md

```diff
@@ -161,7 +161,7 @@ The `{% for %}` tag also works very similarly to how it does in Django. To cycle
 
 ### Other tags
 
-Look into [assign](https://shopify.github.io/liquid/tags/variable/) and see it used [in the sponsors footer](https://github.com/djangocon/2022.djangocon.us/blob/develop/_includes/sponsor-footer.html#L3).
+Look into [assign](https://shopify.github.io/liquid/tags/variable/) and see it used [in the sponsors footer](https://github.com/djangocon/2023.djangocon.us/blob/main/_includes/sponsor-footer.html#L3).
```

Comment:
Explanation for that one: we used `develop` as a staging branch for the page relaunch when switching from either coming soon to CFP is open or CFP closed to schedule revealed in a previous year (might've been 2019?)

Reply:
Thank you for the clarification!